modin/pandas/base.py

```diff
@@ -1477,24 +1551,25 @@ def _stat_operation(
 
         Parameters
         ----------
-        op_name: str,
+        op_name : str
             Name of method to apply.
-        axis: int or axis name,
+        axis : int or axis name,
             Axis to apply method on.
-        skipna: bool,
+        skipna : bool
             Exclude NA/null values when computing the result.
-        level: int or level name,
+        level : int or level name
             If specified `axis` is a MultiIndex, applying method along a particular
-            level, collapsing into a Series
-        numeric_only: bool
+            level, collapsing into a Series.
+        numeric_only : bool, default: None
```

Comment:
```suggestion
        numeric_only : bool, optional
```
I think `optional` suits better when default value is `None` in almost any case